### PR TITLE
Kgpe c bootblock

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: configs/config.asus_kgpe_d16 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CONFIG_VENDOR_ASUS=y
+CONFIG_BOARD_ASUS_KGPE_D16=y
+
```
Comment (macpijan) [inline configs/config.asus_kgpe_d16:3]: We can skip the empyt line I guess

File: src/cpu/amd/Makefile.inc [hunk 2]
```diff
@@ -1,2 +1,3 @@
 subdirs-$(CONFIG_CPU_AMD_AGESA) += agesa
 subdirs-$(CONFIG_CPU_AMD_PI) += pi
+subdirs-$(CONFIG_CPU_AMD_SOCKET_G34_NON_AGESA) += socket_G34
```
Comment (macpijan) [inline src/cpu/amd/Makefile.inc:3]: github complains on missing NL

**Comment by macpijan** [comment]
We may fix the Jenkins complains I guess

```
coreboot@f29383b63c05:~/coreboot$ make what-jenkins-does
Check that files have license headers (lint-stable-000-license-headers): src/cpu/amd/family_10h-family_15h/ram_calc.c has no recognized SPDX identifier.
src/cpu/amd/family_10h-family_15h/ram_calc.h has no recognized SPDX identifier.
src/cpu/amd/socket_G34/socket_G34.c has no recognized SPDX identifier.
src/northbridge/amd/amdfam10/bootblock.c has no recognized SPDX identifier.
test failed
Check for superfluous whitespace in the tree (lint-stable-003-whitespace): success
Check that C labels begin at start-of-line (lint-stable-004-style-labels): success
Check that every board has a meaningful board_info.txt (lint-stable-005-board-status): success
Check that every vendor and board has a Kconfig.name (lint-stable-006-board-name): success
Check Kconfig files for errors (lint-stable-008-kconfig): #!!!!! Error: Undefined Symbol 'CPU_AMD_SOCKET_G34' used at src/cpu/amd/family_10h-family_15h/Kconfig:20.
#!!!!! Error: Undefined Symbol 'CPU_MICROCODE_MULTIPLE_FILES' used at src/cpu/amd/family_10h-family_15h/Kconfig:13.
#!!!!! Error: Undefined Symbol 'PARALLEL_CPU_INIT' used at src/mainboard/asus/kgpe-d16/Kconfig:16.
#!!!!! Error: Undefined Symbol 'SMBUS_HAS_AUX_CHANNELS' used at src/southbridge/amd/sb700/Kconfig:25.
# 4 errors
test failed
```

File: src/cpu/amd/Makefile.inc (hunk 2 above)
Comment (miczyg1) [reply src/cpu/amd/Makefile.inc]: Done

File: configs/config.asus_kgpe_d16 (hunk 1 above)
Comment (miczyg1) [reply configs/config.asus_kgpe_d16]: Done

**Comment by miczyg1** [comment]
All lint cheks now pass

**Pull request merged**